src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    public Square(String color, String name) {
+        setColor(color);
+        setName(name);
+
+    }
+
+    @Override
+    public double getArea() {
+        return super.getArea();

```

Comment:
The getArea() method is supposed to be overridden to provide the specific area calculation for a Square. However, it currently just calls the superclass method which is likely not implemented for the specific case of a square.